requirements.txt

```diff
@@ -4,123 +4,54 @@
 #
 #    pip-compile --no-index --output-file=requirements.txt requirements.in
 #
-azure-core==1.8.2
```

Comment:
You probably need to install a newer version of `pip-tools` - the formatting has changed on one of the recent versions.

Reply:
That's exactly right. Reinstalling `dev-requirements.txt` after rebasing my brach was the thing I've missed. Thanks.